pkgs/development/python-modules/nuclear/default.nix

```diff
@@ -0,0 +1,51 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  colorama,
+  mock,
+  pyyaml,
+  pydantic,
+  backoff,
+  setuptools,
+}:
+
+buildPythonPackage rec {
+  pname = "nuclear";
+  version = "2.2.5";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "igrek51";
+    repo = "nuclear";
+    rev = version;
+    hash = "sha256-JuO7BKmlQE6bWKqy1QvX5U4A9YkKu/4ouTSJh9R7JGo=";
+  };
+
+  build-system = [ setuptools ];
+  propagatedBuildInputs = [
+    colorama
+    pyyaml
+  ];
+
+  nativeCheckInputs = [
+    pytestCheckHook
+    mock
+    pydantic
+    backoff
+  ];
+  pytestFlagsArray = [
+    "tests"
+    "--ignore tests/autocomplete/test_bash_install.py"
+    "--ignore tests/sublog/test_context_logger.py"
```

Comment:
During developing the merge request, one test fails unexpectedly - probably due to how the sandbox handles time zones. I do not have a deep understanding of this and would be happy for any help here.

The issue is described in detail here: https://github.com/igrek51/nuclear/issues/9

To trigger the issue, you have to delete `"--ignore tests/sublog/test_context_logger.py"` in the merge request.

Reply:
Please rephrase these in terms of `disabledTests` and `disabledTestPaths`, with comments as to why they are disabled

Reply:
It seems the test doesn't expect a trailing `Z` in the datetime. Perhaps this is locale dependent. Try something like
```nix
preCheck = ''
  export LC_ALL=C
'';
```

Reply:
I tried the `export LC_ALL=C` trick but it did not have any effect on this test. For now the test is disabled but I will try to dig deeper on this together with the author.